docs/configure.rst

```diff
@@ -0,0 +1,111 @@
+.. highlight:: console
+
+
+.. _configure:
+
+=========
+Configure
+=========
+
+This page shows you how to install `Croud`_, the dedicated command-line
+interface (CLI) and log in to CrateDB Cloud. Croud lets you interact with
+CrateDB Cloud — from creating organizations and projects to deploying clusters.
+
+.. CAUTION::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. NOTE::
+
+  This section assumes that you have `created a CrateDB Cloud account`_.
+
+Install Croud
+=============
+
+Install Croud by running the following command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud is installed, the command returns the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+.. NOTE::
+
+  Make sure to always `update Croud to the latest version`_ to ensure it has all the latest features.
```

Comment:
show the reader how to do that

Reply:
you wanted to link to the instructions in the docs, remember? it initially was described here.